src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -329,8 +367,8 @@ void AtlasEngine::_drawGlyph(const AtlasQueueItem& item) const
             // So for safe measure we'll always scale them to the exact size.
             // But add 1px to the destination size, so that we don't end up with fractional pixels.
             scalingRequired = true;
-            scale.x = (layoutBox.x + _r.dipPerPixel) / boxSize.x;
-            scale.y = (layoutBox.y + _r.dipPerPixel) / boxSize.y;
+            scale.x = layoutBox.x / boxSize.x;

```

Comment:
wat, why'd this need to change again?

Reply:
The extra pixel added to the `layoutBox` causes the vertical lines here:
![image](https://user-images.githubusercontent.com/2256941/182234716-65f97c15-3dcc-4cd9-8c54-b7d74d88ed0e.png)

Without the extra pixel, that issue doesn't occur even if we don't recognize the glyph as a box drawing one.